src/sas/qtgui/Perspectives/SizeDistribution/SizeDistributionLogic.py

```diff
@@ -90,15 +90,15 @@ def fitFlatBackground(self, qmin, qmax):
         """
         Estimate the flat background
         """
-        background = background_fit(self.data, None, qmin, qmax)
-        return background[1]
+        background, background_err = background_fit(self.data, 0, qmin, qmax)
+        return np.exp(background)[0]
```

Comment:
I think ` return self.fitBackgroundScale(0, qmin, qmax)` would be easier to maintain, long-term. If the return of `background_fit` ever changes, it will only need to be updated in one place instead of two.

Reply:
Humm... would it not be easier to just call from SizeDistribution.py? instead of calling fitFlatBackground or fitBackgroundScale just call background_fit in sascalc directly?  

But maybe that is harder to understand? Not good coding practice? simpler would seem better to me?  I suspect this was set up before we really knew how it was all going to work? Just a thought

Reply:
I think what @krzywon is suggesting is that `fitFlatBackground` and `fitBackgroundScale` have the same code and it would be better to unite them (basically the caller should pass the 0 I guess?